File: WatchTvSeries/app/src/main/java/com/example/watchtvseries/viewModel/TvShowViewModel.java
package com.example.watchtvseries.viewModel;

import android.app.Application;

import javax.inject.Inject;

class TvShowViewModel(application:Application) : AndroidViewModel(application){

    fun applyQueries(): HashMap<String, String> {

        val queries: HashMap<String, String> = HashMap()
        queries[QUERY_STR] = "anything"

        return queries
    }
}

File: WatchTvSeries/app/src/main/java/com/example/watchtvseries/viewModel/TvShowViewModel.kt
package com.example.watchtvseries.viewModel

import android.app.Application
import androidx.lifecycle.AndroidViewModel

class TvShowViewModel(application: Application) : AndroidViewModel(application){
    val QUERY_STR : String = "lala"
    fun applyQueries(): HashMap<String, String> {

        val queries: HashMap<String, String> = HashMap()
        queries[QUERY_STR] = "anything"

        return queries
    }
}
